### PR TITLE
add nrk-input

### PR DESCRIPTION
**Pull request by htor**
fixes https://github.com/nrkno/core-css/issues/56

### CODE REVIEW

File: lib/core-css.css [hunk 1]
```diff
@@ -171,6 +172,127 @@ main { display: block }             /* Polyfill main element for IE9/10/11 */
...
+  position: relative;
+  vertical-align: middle;
+  background: white;
+  margin: 0;
+  border: 0;
+  max-width: 100%;
+  outline: 0;
+  color: inherit;
+  font: inherit;
+  line-height: 1.5;
```
Comment (eirikbacker) [inline lib/core-css.css]: Comment why? :)

File: lib/core-css.css [hunk 2]
```diff
@@ -171,6 +172,127 @@ main { display: block }             /* Polyfill main element for IE9/10/11 */
...
+  box-shadow: 0 0 0 1px #e0e0e0;
```
Comment (eirikbacker) [inline lib/core-css.css]: Comment why we use box-shadow instead of border? :)

File: lib/core-css.css [hunk 3]
```diff
@@ -171,6 +172,127 @@ main { display: block }             /* Polyfill main element for IE9/10/11 */
...
+.nrk-input:disabled {
+  cursor: not-allowed !important;
+  opacity: .5;
+}
+
+/* reset ms styles */
+.nrk-input::-ms-clear,
+.nrk-input::-ms-expand,
+.nrk-input[type="radio"]::-ms-check,
+.nrk-input[type="checkbox"]::-ms-check {
```
Comment (eirikbacker) [inline lib/core-css.css]: what happens if just using
`.nrk-input::-ms-check` ? :)

File: lib/core-css.css [hunk 4]
```diff
@@ -171,6 +172,127 @@ main { display: block }             /* Polyfill main element for IE9/10/11 */
...
+  display: none
+}
+
+.nrk-input::-ms-value {
```
Comment (eirikbacker) [inline lib/core-css.css]: Add comment on what it does style ? :)

File: lib/core-css.css [hunk 5]
```diff
@@ -171,6 +172,127 @@ main { display: block }             /* Polyfill main element for IE9/10/11 */
...
+/* reset number styles */
+.nrk-input[type="number"] {
+  -webkit-appearance: none;
+  -moz-appearance: textfield;
+  appearance: textfield;
+}
+
+.nrk-input::-webkit-outer-spin-button,
+.nrk-input::-webkit-inner-spin-button {
+  -webkit-appearance: none;
```
Comment (eirikbacker) [inline lib/core-css.css]: Add comment on what it does style? :)

File: lib/core-css.css [hunk 6]
```diff
@@ -171,6 +172,127 @@ main { display: block }             /* Polyfill main element for IE9/10/11 */
...
+.nrk-input[type="checkbox"],
+.nrk-input[type="radio"] {
+  width: 22px;
+  height: 22px;
+  vertical-align: middle;
+  cursor: pointer;
+  padding: 0;
+}
+
+.nrk-input[type="checkbox"]:checked {
```
Comment (eirikbacker) [inline lib/core-css.css]: Add comment on why we use this technique to draw a check icon? :)

**Comment by mia-aasbakken** [comment]
My nitpicky eyes saw this: 
image 👀

**Comment by htor** [comment]
(windows) pixel bug should be fixed now, tested on win10+chrome.

**Pull request merged**